<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1391</width>
      <height>871</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>901</width>
        <height>801</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="volunteerTableWidget">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>50</y>
        <width>341</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Apellido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Conductor</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="not_confirmed_volunteerTable">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>490</y>
        <width>341</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Apellido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Conductor</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="tableWidgetAllVolunteer">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>11</y>
        <width>401</width>
        <height>791</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Apellido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Conductor</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="labelNameVolunteer">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditName">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLastname1">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLastname2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPhone">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditBirth">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>280</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelName">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>120</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLastname1">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>160</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellido 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLastname2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>200</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellido 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTlf">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>240</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Teléfono</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBirthdate">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>280</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha de nacimiento</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelID">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>120</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NIF/NIE/Pasaporte</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditDNI">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelEmail">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>160</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEmail">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCcaa">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>200</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxProvince">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>240</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxAssembly">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>280</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelCcaa">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Comunidad Autónoma</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelProvince">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>240</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Provincia</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAssembly">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>280</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Asamblea</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCarnet">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>120</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Carnet B</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonCarnetYes">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>120</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sí</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonCarnetNo">
      <property name="geometry">
       <rect>
        <x>1240</x>
        <y>120</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelExp4x4">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>160</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Exp. 4x4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton4x4No">
      <property name="geometry">
       <rect>
        <x>1240</x>
        <y>160</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton4x4Yes">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>160</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sí</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPosition">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>200</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPosition">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>200</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Puesto solicitado</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1399</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionCalendario"/>
    <addaction name="actionLista_voluntarios"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionDocumentaci_n"/>
   </widget>
   <addaction name="menuVer"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDocumentaci_n">
   <property name="text">
    <string>Documentación</string>
   </property>
  </action>
  <action name="actionLista_voluntarios">
   <property name="text">
    <string>Lista voluntarios</string>
   </property>
  </action>
  <action name="actionCalendario">
   <property name="text">
    <string>Calendario</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
